pkgs/development/node-packages/node-packages.json

```diff
@@ -331,5 +331,4 @@
 , "@yaegassy/coc-nginx"
 , "yalc"
 , "yarn"
-, "@zwave-js/server"
```

Comment:
You should also use `pkgs/development/node-packages/remove-attr.py` to remove it from `node-packages.nix`.

Reply:
Done, thanks